src/main/java/uk/gov/hmcts/reform/sscs/robotics/RoboticsJsonMapper.java

```diff
@@ -76,11 +77,11 @@ private static JSONObject buildAppointeeDetails(Appointee appointee, boolean sam
         return buildContactDetails(json, appointee.getAddress(), appointee.getContact());
     }
 
-    private static JSONObject buildJointPartyDetails(JointPartyName jointPartyName, Address jointPartyAddress, boolean sameAddressAsAppellant,
+    private static JSONObject buildJointPartyDetails(Name name, Address jointPartyAddress, boolean sameAddressAsAppellant,
```

Comment:
Doesn't matter much but jointPartyAddress should be just address if its following your pattern you have used above.